[#130] Brew services for Tezos binaries (#149)
serokell/tezos-packaging: README.md

`tezos-{accuser, endorser}` have configurable node address, so that they can be used with both
remote and local node.

### Launchd services on macOS

`tezos-accuser-<proto>`, `tezos-baker-<proto>`, `tezos-endorser-<proto>` formulas

> provide backround services for running corresponding daemons.

Reviewer: ```suggestion
provide backround services for running the corresponding daemons.
```

> Since `tezos-node` and `tezos-signer` provide multiple services they are provided
> in a dedicated meta-formulas. These formulas doesn't install any binaries and only provide
> backround services.

Reviewer: ```suggestion
Since `tezos-node` and `tezos-signer` need multiple services they are provided
in dedicated meta-formulas. These formulas don't install any binaries and only add
background services.
```
A couple of typos and I tried to not repeat "provide" too much.

Formulas with `tezos-node` background services:
* `tezos-node-mainnet`
* `tezos-node-edo2net`

Formulas with `tezos-signer` background services:
* `tezos-signer-http`
* `tezos-signer-https`
* `tesos-signer-tcp`
* `tezos-signer-unix`

To start the service: `brew services start <formula>`.

> To stop the services: `brew services stop <formula>`.

Reviewer: ```suggestion
To stop the service: `brew services stop <formula>`.
```
nitpick, but this way it matched the line above.

All of the brew services have various configurable env variables. These variables

> can be changed in the corresponding `usr/local/Cellar/tezos-signer-tcp/<version>/homebrew.mxcl.<formula>.plist`.

Reviewer: Is there no easier way to modify these files, e.g. a launchd `edit` option?

Author: I suppose, there is a GUI tool, but I can't check, because I don't have mac with GUI :cry:

Author: So, AFAIU, there is no anything related to `edit` in both `launchctl` and `brew services` and the suggested way is to use any XML editor (or just any text editor)
